operator/params.yaml

```diff
@@ -30,6 +30,26 @@ NODE_STORAGE_CLASS:
   description: "The storage class to be used in volumeClaimTemplates. By default, it is not required and the default storage class is used."
   required: false
 
+NODE_READINESS_PROBE_INITIAL_DELAY_S:
```

Comment:
Is the `_S` necessary if the description is already clear? Same question for period and timeout.